deploy/config/cpe_themis.yaml

```diff
@@ -0,0 +1,63 @@
+---
+  # Each server can be configured using the following options:
+  # https://godoc.org/github.com/xmidt-org/themis/xhttp/xhttpserver#Options
```

Comment:
Since the Go docs throughout our code is pretty thorough I thought it'd be easiest to provide a link to them here in configuration. This way, whoever is working on configuring their themis can have an easy time figuring out the meaning behind the config values / options

Reply:
Personally, I think having a link as a supplement is nice but doesn't quite replace actually commenting each value with a small description.  We don't have to go into a lot of detail, but part of the benefit of the comments here is that anyone who installs the rpm can go read the documentation in this file to determine how to set up the service.  I'm not sure a link is as convenient?

I think this is fine for now though.

Reply:
I think this is fine for now too as having a thorough comment-based config deserves its own PR.